src/pages/Integrations/Create/IntegrationsStep.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  componentTypes,
+  validatorTypes,
+} from '@data-driven-forms/react-form-renderer';
+import { compileAllIntegrationComboOptions, iconMapper } from './helpers';
+import { CARD_SELECT } from './CreateWizard';
+import { defaultIconList } from '../../../config/Config';
+
+export const integrationsStep = (category: string) => ({
+  title: 'Select Integration type',
+  name: 'integrationType',
+  nextStep: 'details',
+  fields: [
+    {
+      component: componentTypes.PLAIN_TEXT,
+      name: 'plain-text',
+      label:
+        'Select a communications integration to add to your Red Hat account.',
+    },
+    {
+      component: [CARD_SELECT],
```

Comment:
No need to pass in array of single string.
```suggestion
      component: CARD_SELECT,
```